schema/beneficial-ownership-statements.json

```diff
@@ -1,8 +1,15 @@
 {
```

Comment:
For the time being, I think we should drop beneficial-ownership-statement**s**.json (plural) as this doesn't fit the model in #57.